en-US/Players_Guide/03_Creating_a_Character/Equipment/Adventuring_Gear.md

```diff
@@ -4,5 +4,303 @@ This section describes items that have special rules or require further explanat
+- **Poison, Basic.**
+  You can use the poison in this vial to coat one slashing or piercing weapon or up to three pieces of ammunition.
+  Applying the poison takes an action.
+  A creature hit by the poisoned weapon or ammunition must make a DC 10 Constitution saving throw or take `1d4` poison damage.
+  Once applied, the poison retains potency for 1 minute before drying.
+- **Potion of Healing.**
+  A character who drinks the magical red fluid in this vial regains `2d4 + 2` hit points.
+  Drinking or administering a potion takes an action.
+- **Pouch.**
+  A cloth or leather pouch can hold up to 20 sling bullets or 50 blowgun needles, among other things.
+  A compartmentalized pouch for holding spell components is called a component pouch (described earlier in this section).
+- **Quiver.**
+  A quiver can hold up to 20 arrows.
+- **Ram, Portable.**
+  You can use a portable ram to break down doors.
+  When doing so, you gain a `+4` bonus on the Strength check.
+  One other character can help you use the ram, giving you advantage on this check.
+- **Rations.**
+  Rations consist of dry foods suitable for extended travel, including jerky, dried fruit, hardtack, and nuts.
+- **Rope.**
+  Rope, whether made of hemp or silk, has 2 hit points and can be burst with a DC 17 Strength check.
+- **Scale, Merchant’s.**
+  A scale includes a small balance, pans, and a suitable assortment of weights up to 2 pounds.
+  With it, you can measure the exact weight of small objects, such as raw precious metals or trade goods, to help determine their worth.
+- **Spellbook.**
+  Essential for Wizards, a spellbook is a leather-bound tome with 100 blank vellum pages suitable for recording spells.
+- **Spyglass.**
+  Objects viewed through a spyglass are magnified to twice their size.
+- **Tent.**
+  A simple and portable canvas shelter, a tent sleeps two.
+- **Tinderbox.**
+  This small container holds flint, fire steel, and tinder (usually dry cloth soaked in light oil) used to kindle a fire.
+  Using it to light a torch—or anything else with abundant, exposed fuel—takes an action.
+  Lighting any other fire takes 1 minute.
+- **Torch.**
+  A torch burns for 1 hour, providing bright light in a 20-foot radius and dim light for an additional 20 feet.
+  If you make a melee attack with a burning torch and hit, it deals `1` fire damage.
+- **Wodewose Focus.**
+  A Wodewose focus might be a sprig of mistletoe or holly, a wand or scepter made of yew or another special wood, a staff drawn whole out of a living tree, or a totem object incorporating feathers, fur, bones, and teeth from sacred animals.
+  A Wodewose can use such an object as a spellcasting focus.
+
+___
+<!-- markdownlint-disable-next-line no-emphasis-as-heading -->
+_Adventuring Gear (table)_
+
+| Item                              |   Cost |     Weight |
+|:----------------------------------|-------:|-----------:|
+| Abacus                            |   2 gp |      2 lb. |
```

Comment:
This isn't in the Google Doc, but it is in the WotC SRD 5.1. I'm guessing it got lost during the copying process?